src/lib/components/table/QTable.svelte

```diff
@@ -96,19 +98,23 @@
       type: !sort?.type || sort?.type === "desc" ? "asc" : "desc",
     };
   }
+
+  Q.classes("q-table__table", {
+    bemClasses: {
+      flat,
+      bordered,
+      dense,
+    },
+    classes: [props.class],
+  });
 </script>
 
-<div class="q-table" {...$$restProps}>
-  <table
-    class="q-table__table {userClasses}"
-    class:q-table--flat={flat}
-    class:q-table--bordered={bordered}
-    class:q-table--dense={dense}
-  >
+<div {...props} class="q-table">
+  <table class="q-table__table" {...Q.classes}>
```

Comment:
Do we really need to wrap the table in a div?

Reply:
Yes, what's the alternative? There's a row selection at the bottom.